src/Metrics.tcl

```diff
@@ -254,5 +254,250 @@ proc report_design_area_metrics {args} {
   utl::metric_float "design__instance__utilization__stdcell" $stdcell_util
 }
 
+proc report_puts { out } {
+  upvar 1 when when
+  upvar 1 filename filename
+  set fileId [open $filename a]
+  puts $fileId $out
+  close $fileId
+}
+
+define_cmd_args "report_metrics" {[-stage][-when][-include_erc][-include_clock_skew][-metrics_report_dir dir]}
+proc report_metrics { stage when {include_erc true} {include_clock_skew true} {metrics_report_dir "."}} {
+  if {[info exists ::env(SKIP_REPORT_METRICS)] && $::env(SKIP_REPORT_METRICS) == 1} {
```

Comment:
Should this be here? I think this should be removed.

Reply:
Am I correct in understanding that checking these variables here makes no sense, and we need to remove these pieces of code from this part?

Reply:
Correct, the command as a whole should be skipped at the ORFS level based on these vars.